dynamic_rest/serializers.py

```diff
@@ -346,6 +361,22 @@ def get_fields(self):
         for name in deferred:
             serializer_fields.pop(name)
 
+        # Set read_only flags based on read_only_fields meta list.
+        # Here to cover DynamicFields not covered by DRF.
+        ro_fields = getattr(self.Meta, 'read_only_fields', [])
+        self.make_fields_read_only(serializer_fields, ro_fields, True)
+
+        # Toggle read_only flags for immutable fields.
+        immutable_field_names = self._get_flagged_field_names(
+            serializer_fields,
+            'immutable'
+        )
+        self.make_fields_read_only(
```

Comment:
That makes sense. So we're saying that immutable + read-only is an acceptable combination, and immutable will override read-only on POST requests. Sounds reasonable, but it would be nice to have a comment explaining this case.